javascript/exercises/cashRegisterSolution.js

```diff
@@ -0,0 +1,50 @@
+var carts = {
+  brenda: { 
+    banana: "1.25",
+    handkerchief: ".99",
+    Tshirt: "25.01"
+    },
+  claire: {
+    "apple": "0.60",
+    "nalgene": "10.34",
+    "protein_shake": "22.36"
+    }
+};
+
+
+function cashRegister(cart){
+  var people = Object.keys(cart); // array of the carts keys, which are people  
+  var items = []; // array of the each people keys, which are items  
+  var result = [];
+  
+  for (var i = 0; i < people.length; i++) {
+    // for each person grab the every item
+    items[i] = Object.keys(cart[people[i]]); 
+  }
+
+  var i = 0;
+  var j = 0;
+  var sum = 0;
+
+  while(i < people.length){
+  // grab the price of every item for a particular person 
+    // person is represented by cart[people[i]]
```

Comment:
This is a nice break down, but requires changing the logic inside the while loop. I'll leave this for another time. Instead of changing the solution, I added some notes.